engine/auxiliary.py

```diff
@@ -2,7 +2,7 @@
 import inspect, os
 import logging
 logger = logging.getLogger('nipype.workflow')
-from nipype import Node
+from .node import Node
```

Comment:
This is the `Node` current nipype and I don't have `node.py` 

This is only used to my wrapper for the current interfaces. I'll work now to connect the current interfaces directly, but for now it `CurrentInterface` is using nipype.Node